lua/apisix/plugins/limit-count/limit-count-redis.lua

```diff
@@ -20,6 +20,7 @@ local resty_lock = require("resty.lock")
 local assert = assert
 local setmetatable = setmetatable
 local tostring = tostring
+local options = {pool_size = 1000, backlog = 1000}
```

Comment:
hard code options?

Reply:
> hard code options?

Maybe I can specify pool_size and backlog from plugin ` limit-count`?

Reply:
yes, that is much better.

Reply:
> hard code options?

Done